libp2p.nimble

```diff
@@ -31,9 +31,9 @@ proc runTest(filename: string, moreoptions: string = "") =
   if getEnv("CICOV").len > 0:
     excstr &= " --nimcache:nimcache/" & filename & "-" & $excstr.hash
   exec excstr &
-    " -r -d:libp2p_quic_support -d:libp2p_autotls_support -d:libp2p_mix_experimental_exit_is_dest tests/" &
+    " -r -d:libp2p_quic_support -d:libp2p_autotls_support -d:libp2p_mix_experimental_exit_is_dest -d:libp2p_gossipsub_1_4 tests/" &
     filename
-  rmFile "tests/" & filename.toExe
+  #rmFile "tests/" & filename.toExe
```

Comment:
why is this commented? remove if it needs to be removed.

Reply:
My bad, leftovers from testing